lib/till.rb

```diff
@@ -0,0 +1,45 @@
+class Till
+
+  def self.coffee_shop=(shop)
+    @coffee_shop = shop
+  end
+
+  def self.customer_order=(order)
```

Comment:
Style/TrivialAccessors: Use `attr_writer` to define trivial writer methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)